src/app/dataset-flow/dataset-flow-details/dataset-flow-details.types.ts

```diff
@@ -0,0 +1,23 @@
+import {
+    DatasetBasicsFragment,
+    DatasetPermissionsFragment,
+    FlowHistoryDataFragment,
+    FlowSummaryDataFragment,
+} from "src/app/api/kamu.graphql.interface";
+
+export enum FlowDetailsTabs {
+    SUMMARY = "summary",
+    HISTORY = "history",
+    LOGS = "logs",
+    USAGE = "usage",
+    ADMIN = "admin",
+}
+
+export interface DatasetFlowByIdResponse {
+    flow: FlowSummaryDataFragment;
+    flowHistory: FlowHistoryDataFragment[];
+}
+export interface ViewMenuData {
```

Comment:
Blank line. I'm surprised Prettier is not inserting it.

Reply:
Fixed